Gruntfile.js

```diff
@@ -56,6 +56,16 @@ module.exports = function (grunt) {
           '--ignore=src/webgpu/listing.ts',
         ],
       },
+      'build-out-node': {
```

Comment:
using `tsc` instead of `babel`, so we don't have the generated comment at the top of the file. Is that okay? If Chromium doesn't want a dep on babel, we would lose the generated comment in /gen/..wpt/ too. But, it would be in /gen so I think it should be fairly clear.

Reply:
I think that's fine since we aren't checking in the generated code.